sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -618,6 +645,7 @@ def test_partial_dependence_dataframe(estimator, preprocessor, features):
         assert_allclose(values_pipe[1], values_clf[1])
 
 
+@pytest.mark.filterwarnings("ignore:A Bunch will be returned")
```

Comment:
This is the last change in the file, so I am mentioning here. We need an additional test to check that we properly raise the FutureWarning with the legacy option.

Reply:
It could be good to add a test to check that `kind="average"` is actually the average of `kind="individual"`.

Reply:
Done in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/0fe1c8bd2a8d40acc864e12edd98180ecd44170c and https://github.com/scikit-learn/scikit-learn/pull/16619/commits/0a88099e5bb4c34a40949cae2fdf3752437389fd